include/seastar/util/backtrace.hh

```diff
@@ -59,13 +61,15 @@ frame decorate(uintptr_t addr) noexcept;
 SEASTAR_MODULE_EXPORT
 template<typename Func>
 void backtrace(Func&& func) noexcept(noexcept(func(frame()))) {
+#ifdef Seastar_HAVE_EXECINFO
     constexpr size_t max_backtrace = 100;
     void* buffer[max_backtrace];
     int n = ::backtrace(buffer, max_backtrace);
     for (int i = 0; i < n; ++i) {
         auto ip = reinterpret_cast<uintptr_t>(buffer[i]);
         func(decorate(ip - 1));
     }
+#endif
```

Comment:
I wonder if we shouldn't in an `#else` return some sort of string saying backtrace() is unimplemented. Or at least leave a "FIXME" comment. One day we'll need to fix that, right?

Reply:
By the way, a question about the preexisting code: Why is backtrace() defined in the header file? Do end users use it directly, or things implemented with it? Even if end-users do use it directly, it should switch to taking std::function (or seastar::noncopyable_function) instead of being a template - and then the implementation can be moded to the source file. It's not important for performance.